main.pm

```diff
@@ -2,6 +2,8 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 
 use strict;
+use warnings;
```

Comment:
This is not following what http://open.qa suggests. If you think we have a general shortcoming in missing warnings handling in os-autoinst please address that on the according upstream level first, not here downstream.

Reply:
As we found in the Unblock https://open.qa/docs/#_test_module_interface uses `use Mojo::Base "basetest";` so it's not directly comparable.

Most likely it would be best to drop the change from this PR/branch.

Reply:
Raised os-autoinst/openQA#5404 from the discussion.